internal/types/studies.go

```diff
@@ -58,6 +58,7 @@ type StudyAdmin struct {
 
 type Asset struct {
 	ModelAuditable
+	UserID                 uuid.UUID `gorm:"not null;index"`
```

Comment:
I wonder if something like `CreatorUserID` would be better. or is it obvious that `asset.UserID` is the person who created it?